src/components/ProjectSearch/TextSearch.js

```diff
@@ -68,7 +77,7 @@ export default class TextSearch extends Component {
     return dom.div(
       {
         className: classnames("result", { focused }),
-        onClick: () => this.selectMatchItem(match)
+        onClick: () => setTimeout(() => this.selectMatchItem(match), 50)
```

Comment:
On clicking a match, the page opens so quickly you have no visual indication of what you clicked. This just allows, the user see a visual indication for a few seconds before the source file is opened